src/renderers/dom/shared/hooks/ReactDOMUnknownPropertyHook.js

```diff
@@ -67,6 +67,11 @@ if (__DEV__) {
     if (EventPluginRegistry.registrationNameModules.hasOwnProperty(name)) {
       return true;
     }
+    if (EventPluginRegistry.plugins.length === 0 && name.indexOf('on') === 0) {
```

Comment:
Yeah, it's probably not enough of a difference to actually matter. And it's dev-only anyway I guess.

https://jsperf.com/asdiohsd